cobra/io/sbml.py

```diff
@@ -1238,8 +1254,8 @@ def _sbase_notes_dict(sbase, notes):
 
 In the current stage the new annotation format is not completely supported yet.
 """
-URL_IDENTIFIERS_PATTERN = r"^http[s]{0,1}://identifiers.org/(.+)/(.+)"
-URL_IDENTIFIERS_PREFIX = r"https://identifiers.org"
+URL_IDENTIFIERS_PATTERN = re.compile(r"^http[s]{0,1}://identifiers.org/(.+?)/(.+)")  # noqa: E501
```

Comment:
I was wondering, do we really need a regex here? How about [`urlsplit`](https://docs.python.org/3/library/urllib.parse.html?highlight=urlsplit#urllib.parse.urlsplit)? (Careful, it has a different location in Python 2.) If you do feel that the regex is easier I suggest below change.

```suggestion
URL_IDENTIFIERS_PATTERN = re.compile(r"^https?://identifiers.org/(.+?)/(.+)")  # noqa: E501
```